### ISSUE TITLE
Bash Powerline causing view issues with HyperTerm

### ISSUE THREAD

**Issue by jackyliang** [zeit/hyper#941]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate
- **OS version and name**: Mac OS X 10.11.6 (15G1004)
- **Hyper.app version**: _how do I check? I installed this yesterday, so probably 0.8.3_
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: Everything is the same except for the addition of:

```
  plugins: [                                                                                               
    'hyperterm-monokai',                                                                                   
    'hyper-blink',                                                                                         
    'hyperterm-cursor'                                                                                     
  ],  
```
- **The issue is reproducible in vanilla Hyper.app**: nope.
## Issue

I have Powerline installed on my default Bash shell. It's activated within my bash profile using the following line: `source ~/.bash-powerline.sh`

However, this seems to be causing issues with Hyper, because:
1. No matter what theme I install, if I comment out `source ~/.bash-powerline.sh`, there is no coloring on the state of my git repo. What I mean by this is, let's say you check out [hyperpunk](https://www.npmjs.com/package/hyperpunk) theme - where it shows you the path of your current directory and what branch you're on, it's color coded. However, if I disable bash powerline, that line is just a single color.
2. The Powerline line is slightly obscured, so if I type in an underscore, it's blocked by something, like ![this](https://puu.sh/rXEAZ/edc4aa9f59.png). The branch name has underscores in it, but it's obscured. 

I tried to fix this by uninstalling Powerline, however, issue #1 persists where there's no color coding in the states of my branch or the location of my current directory. However, if I keep Powerline, the line is slightly obscured..

I also tried playing with the padding, with increasing it, decreasing it, or just straight up removing it, but it's still causing issues.

**Comment by Stanzilla** [comment]
Hey there, please try again with the new Hyper v2 release and open a new issue if the problem stills exists. Thank you!
